content/applications/inventory_and_mrp/manufacturing/management/work_center_time_off.rst

```diff
@@ -0,0 +1,117 @@
+developer mode` under the **Developer Tools** heading.
+
+.. image:: work_center_time_off/developer-mode-button.png
+   :align: center
+   :alt: The "Activate the developer mode" button.
+
+Next, install the :guilabel:`Time Off` app. This is the app used for assigning time off to all
+resources within Odoo, including employees and work centers. Navigate to :guilabel:`Apps`, then type
+"Time Off" in the :guilabel:`Search...` bar. The card for the :guilabel:`Time Off` module should be
+the only one that appears on the page. Click the green :guilabel:`Install` button on the card to
+install the app.
+
+.. image:: work_center_time_off/time-off-install-card.png
+   :align: center
+   :alt: The Time Off module installation card.
+
+The last step is to properly configure work centers. For this workflow, it is necessary to have at
+least two work centers: one that will be made unavailable and a second that will receive the work
+orders that the other cannot accept. If no second work center is configured, Odoo will not be able
+to route work orders away from the unavailable work center and they will pile up in its queue.
+
+To create a work center, navigate to :menuselection:`Manufacturing --> Configuration -->
+Work Centers --> Create`. Make sure that both work centers have the same equipment listed under the
+:guilabel:`Equipment` tab. This will ensure that operations carried out at one work center can also
+be performed at the other.
+
+.. image:: work_center_time_off/work-center-equipment-tab.png
+   :align: center
+   :alt: The equipment tab on a work center form.
+
+For the work center that will be made unavailable, select the second work center on the
+:guilabel:`Alternative Workcenters` drop-down. Now, Odoo knows to send work orders to the second
+work center when the first is unavailable for any reason.
+
+.. image:: work_center_time_off/alternative-work-center-selection.png
+   :align: center
+   :alt: A work center form configured with an alternative work center.
+
+Add time off for a work center
+------------------------------
+
+With configuration completed, time off can now be assigned to the work center that will be made
+unavailable. Begin by navigating to :menuselection:`Configuration --> Work Centers` and selecting
+the affected work center. Click :guilabel:`Edit` and then the :guilabel:`External link` button next
+to the :guilabel:`Working Hours` drop-down.
+
+.. image:: work_center_time_off/working-hours-button.png
+   :align: center
+   :alt: The Working Hours "External link" button on the work center form.
+
+A pop-up will appear, titled :guilabel:`Open: Working Hours`. The standard working hours for the
+work center will be listed here, along with various other details about it. Since developer mode was
+enabled, there should be a :guilabel:`Time Off` button in the top right of the pop-up. Click it to
+be taken to the :guilabel:`Resource Time Off` page.
```

Comment:
```suggestion
A pop-up window appears, titled :guilabel:`Open: Working Hours`. The standard working hours for the
work center are listed here, along with various other details about it. Since developer mode is
enabled, there is a :guilabel:`Time Off` button in the top right of the pop-up window. Click it to
be taken to the :guilabel:`Resource Time Off` page.
```